components/GlobalStyles.js

```diff
@@ -2,16 +2,23 @@ import { createGlobalStyle } from "styled-components";
 
 const GlobalStyles = createGlobalStyle`
 @font-face {
-    font-family: "JosefinSans";
-    src: url("/fonts/Grace-Regular.ttf");
+   
+
+    font-family: "oswald";
+    src: url("/fonts/Oswald-VariableFont_wght.ttf");
        
+}
+
+@font-face {
+    font-family: "Caveat";
+    src: url("/fonts/Caveat-VariableFont_wght.ttf");
 }
     html,
     body {
       
         padding: 0;
         margin: 0;
-        font-family: "JosefinSans";
```

Comment:
If you have the time, you could add you font as variable to your globalstyles at root. Same with color or backgound-color